src/modules/common/components/__tests__/Breadcrumb.spec.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
```

Comment:
This file should be named Breadcrumb_s_.spec.js

Reply:
Done